worlds/stardew_valley/test/stability/TestStability.py

```diff
@@ -26,8 +26,8 @@ def test_all_locations_and_items_are_the_same_between_two_generations(self):
         output_a = subprocess.check_output([sys.executable, '-m', 'worlds.stardew_valley.test.stability.StabilityOutputScript', '--seed', str(seed)])
         output_b = subprocess.check_output([sys.executable, '-m', 'worlds.stardew_valley.test.stability.StabilityOutputScript', '--seed', str(seed)])
 
-        result_a = json.loads(output_a)
-        result_b = json.loads(output_b)
+        result_a = json.loads(output_a[:-4])
```

Comment:
Can you explain this magic number? Ideally in the code itself with a named constant or a comment?

Reply:
For some reason, 4 bytes appeared at the end of the subprocess output. There is something somewhere that print, but investigating that is definitely out of scope for this PR. (It was already broken in main)

Reply:
I added a comment to explain that tho.